Cargo.toml

```diff
@@ -79,6 +79,7 @@ unicode-segmentation = "1.10.1"
 wayrs-client = { version = "1.0", features = ["tokio"] }
 wayrs-protocols = { version = "0.14", features = ["wlr-foreign-toplevel-management-unstable-v1"] }
 zbus = { version = "5", default-features = false, features = ["tokio"] }
+x11rb-async = { version = "0.13.1", features = ["xkb", "allow-unsafe-code"] }
```

Comment:
I _think_ the `allow-unsafe-code` feature is not used here, needlessly adds a dependency on `lidxcb` and can be removed?

Reply:
oops! right!